src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -139,11 +142,28 @@ protected virtual void Initialize()
             types.Add(type);
 
             ScriptConfig.HostConfig.TypeLocator = new TypeLocator(types);
-            ScriptConfig.HostConfig.NameResolver = new NameResolver();
+
+
+
+            GetConfigFromBindings(functions, ScriptConfig.HostConfig);
 
             Functions = functions;
         }
+        
+        // Bindings may require us to update JobHostConfiguration. 
+        private void GetConfigFromBindings(Collection<FunctionDescriptor> functions, JobHostConfiguration hostConfig)
```

Comment:
Recommend renaming to ApplyBindingConfiguration, matching the existing ApplyConfiguration